### ISSUE TITLE
LT&RT Not working in version 4!

### ISSUE THREAD

**Issue by DDiren** [x360ce/x360ce#1397]
Hello!
My LT and RT buttons do not work in the version 4. They work when I assign them to analog sticks, but when I assign them to corresponding buttons (button 7 & 8), they won't work. Games, in this case Elden Ring, do not see it when I press it. Do you have any solutions to this problem?

**Comment by lazollz** [comment]
I have the same problem. Were you able to find any solution ?

**Comment by Abohanfi** [comment]
[](url)
[x360ce_x64.zip](https://github.com/x360ce/x360ce/files/10702275/x360ce_x64.zip)
![Screenshot 2023-02-10 010321](https://user-images.githubusercontent.com/124939754/217959189-deaff380-f6b3-4da7-81c6-122edc2a61b4.jpg)
pun this in the folder of game after installation

**Comment by Abohanfi** [comment]
that's work for me on win 11

**Comment by DDiren** [comment]
> I have the same problem. Were you able to find any solution ?

Nope, I just tested different games. In some it works, in others it doesn't. For example, I could make it work in NieR Automata but not in Elden Ring or Dark Souls 3.

**Comment by EvoJocys** [comment]
This looks like it was addressed. From the changelog:

4.17.51 — Fixed: Trigger dead zone and anti-dead zone used the wrong range, flattening the curve.
4.20.43 — Fixed: A D-Pad dead zone of 0 % was ignored and used as 100 %.

Please try 4.21: https://github.com/x360ce/x360ce/releases

Closing as part of a tracker clean-up after the 4.21 release. If it still happens on 4.21, reopen this issue with the version and what you see.
